src/main/java/hu/dungeonhunter/characters/Champion.java

```diff
@@ -1,52 +1,50 @@
 package hu.dungeonhunter.characters;
 import hu.dungeonhunter.tools.Dice;
+import lombok.Getter;
+import lombok.Setter;
+
 public class Champion {
 
     static final int START_HP = 20;
-    private int championHP;
-    private final int championMaxDamage;
-    private int championNumOfDices;
-    private boolean championVictory;
 
-    public Champion() {
-        this.championHP = START_HP;
-        this.championMaxDamage = 6;
-        this.championNumOfDices = 1;
-        championStartValues();
-        this.championVictory = false;
-    }
+    @Setter
+    @Getter
+    private int hp;
 
-    public boolean isChampionVictory() {
-        return championVictory;
-    }
+    @Getter
+    private final int maxDamage;
 
-    public void setChampionVictory(boolean championVictory) {
-        this.championVictory = championVictory;
-    }
+    @Setter
+    @Getter
+    private int numOfDices;
 
-    void championStartValues() {
-        System.out.println("Your champion have " + championHP + " HP and can do " +
-                championNumOfDices + "-" + championNumOfDices * championMaxDamage +
-                " damage.");
-    }
+    @Setter
+    @Getter
+    private boolean defeat;
 
-    public int getChampionHP() {
-        return championHP;
+    public Champion() {
+        this.hp = START_HP;
+        this.maxDamage = 6;
+        this.numOfDices = 1;
+        startValues();
+        this.defeat = false;
```

Comment:
No need to give false initializer value to a boolean. When you say `private boolean defeat;` at line 23 it already has false value.